Source/Streamstone.Tests/Scenarios/Tracking_entity_changes.cs

```diff
@@ -577,15 +586,21 @@ yield return
             }
         }
 
-        void InsertTestEntity(TestEntity entity)
+        void InsertTestEntity(ITableEntity entity)
```

Comment:
Ok, disregard my comment. Testing in-memory merge is silly, since we will deal with the same instance of entity being it `DynamicTableEntity` or not, so there is nothing to merge. Only testing at storage level make sense here.